integrations/google-analytics/test/index.test.js

```diff
@@ -1341,6 +1341,45 @@ describe('Google Analytics', function() {
           analytics.assert(window.ga.args[1][0] === 'set');
         });
 
+        it('should map user-defined position properties for products in products array', function() {
```

Comment:
Can we also add a similar test for "Product List Filtered"?

Reply:
Done